### ISSUE TITLE
No rollover or settings after new vault for mobile

### ISSUE THREAD

**Issue by LucBetbeder** [shichongrui/obsidian-rollover-daily-todos#19]
The plugin was working fine (on MacOs Desktop) 
Set up new Desktop vault per iCloud sync instructions for iOS Mobile App.
Copied old files to new vault.  (Path change)
Re-added plugins and now no rollover function (Desktop and iOS) and settings not visible.

**Comment by lennart-h** [comment]
Having the same issue under Ubuntu but never got it working in the first place.

**Comment by acorazza** [comment]
It doesn't work for me at all (Windows)

**Comment by bnmnetp** [comment]
This is what I see in the developer console:

```
Uncaught (in promise) TypeError: Cannot read property 'daily-notes' of undefined
    at RolloverTodosPlugin.checkDailyNotesEnabled (eval at <anonymous> (app.js:1), <anonymous>:9:51)
    at RolloverTodosPlugin.onload (eval at <anonymous> (app.js:1), <anonymous>:42:13)
```

**Comment by jnoebauer** [comment]
Same here. It doesn't work for me at Windows. The settings are not visible.

**Comment by ackstorm23** [comment]
Same here. Doesn't work at all.

**Comment by Narbulus** [comment]
I put out a fix for this in [this PR](https://github.com/shichongrui/obsidian-rollover-daily-todos/pull/21).

**Comment by shichongrui** [comment]
Released as version 1.0.3
